api/TextMetrics.json

```diff
@@ -33,7 +33,7 @@
             "version_added": "3.1"
           },
           "safari_ios": {
-            "version_added": true
+            "version_added": "3"
```

Comment:
This doesn't match https://developer.mozilla.org/en-US/docs/Web/API/CanvasRenderingContext2D/measureText#browser_compatibility and I'm pretty sure the two should be the same version. Can you check double check the `measureText()` data and see if they really do disagree?

If `TextMetrics` was exposed before `measureText()`, then most likely it was just a missing flag in the IDL and accidental. Since that `TextMetrics` interface couldn't be used for anything, I think we should then still use the version from `measureText()`.

Reply:
This was from manual mirroring, and the `width` property is also set to iOS 3.2, so I trust the version number for `measureText()` more.  I'm unable to test anything earlier than Safari 4 at the moment either, so I don't have a way to verify whether this was actually exposed in Safari 3.1 or not.  (Both instances where "3.1" shows up came from the initial wiki migration.)